### DISCUSSION TITLE
Выход из наблюдения за другим ксеном

### DISCUSSION THREAD

**Discussion by Shootnik01** [Cosmic-Overlord/TerraGov-Marine-Corps#299] [Ideas]
Сделать возможным выход из наблюдения за другим ксеном отдельной кнопкой, чтобы можно было в ресте выходить из наблюдения

**Comment by homexp13** [comment]
можно просто ещё раз глазок на том же ксене нажать и всё. скилл ишуй

**Reply by Shootnik01** [reply]
А, рил. Ментор момент
